open-search/upload.yaml

```
@@ -0,0 +1,94 @@
+  labels:
+    app.kubernetes.io/version: "v1.0.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.24.x"
+    tekton.dev/categories: data
+    tekton.dev/tags: "data, results"
+    tekton.dev/displayName: "opensearch import"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: |
+    Task to import junit results into report portal
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)